docs/en/authorization_endpoint.rst

```diff
@@ -195,57 +172,35 @@ When the **scope** parameter is used, the following values are supported:
     - *email*;
     - *email_verified*.
 
-The attributes requested by the parameter **scope** are available both in the ID Token and in the response to the Userinfo Endpoint.
+.. admonition:: |spid-icon|
 
```

Comment:
Only for SPID: if a scope is requested other than 'openid', the RP metadata SHOULD contains id_token_encrypted_response_alg and id_token_encrypted_response_enc and their value SHOULD NOT be 'none'.
If a scope is requested other than 'openid' and the RP metadata do not contains id_token_encrypted_response_alg and id_token_encrypted_response_enc or their value is 'none', none user attributes are returned in the id_token from token endpoint response.

Reply:
I'll keep this comment and try to include it after the next commits and revisions, so please keep this on hold

Reply:
I'd just defined in the metadata parameter definition that the allowed values MUST be the same as the ones adopted for encrypting userinfo ... just a pointer without a re-definition

do you agree?

Reply:
yes, as in SPID Notice 41

Reply:
here my proposal

https://github.com/italia/spid-cie-oidc-docs/issues/82

so if you agree we may merge it as it is and then create a separate PR for that